openssl-sys/build.rs

```diff
@@ -15,10 +15,13 @@ fn main() {
         // rustc doesn't seem to work with pkg-config's output in mingw64
         if !target.contains("windows") {
             if let Ok(info) = pkg_config::find_library("openssl") {
-                // avoid empty include paths as they are not supported by GCC
+                // if we did not find any include paths try to find some other
+                // ones we might want to emit.
                 if info.include_paths.len() > 0 {
                     let paths = env::join_paths(info.include_paths).unwrap();
                     println!("cargo:include={}", paths.to_str().unwrap());
+                } else {
+                    find_fallback_headers();
```

Comment:
JFTR. I verified that if you force link via brew, it spits out include paths and does not go into the fallback case.